scripts/build_renewable_profiles.py

```diff
@@ -240,7 +240,7 @@ def get_hydro_capacity_annual_hydro_generation(
             fn,
             comment="#",
             keep_default_na=False,
-            na_values=["-"],
+            na_values=["-",""],
```

Comment:
@davide-f small bug fix that caused issues. 
Scaling on "hydro_capcities" did not work before.

Will add in a moment a CI with the option 'hydro_capcity' and one with 'eia'